problem-gson/src/test/java/org/zalando/problem/GsonProblemTest.java

```diff
@@ -0,0 +1,296 @@
+package org.zalando.problem;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.stream.JsonReader;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.net.URI;
+import java.net.URL;
+import java.util.List;
+import java.util.Objects;
+
+import static com.jayway.jsonassert.JsonAssert.with;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.hasToString;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.Matchers.startsWith;
+import static org.hobsoft.hamcrest.compose.ComposeMatchers.hasFeature;
+import static org.zalando.problem.Status.BAD_REQUEST;
+
+class GsonProblemTest {
+
+    private static Gson gson;
+
+    @BeforeAll
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/problem/pullRequest?prid=4476243)